src/test/java/org/fundacionjala/at15/katas/poker/diana/TwoPairTest.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.at15.katas.poker.diana;
+
+import org.fundacionjala.at15.katas.pokerhands.diana.TwoPair;
+import static org.junit.Assert.assertTrue;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class TwoPairTest {
+    @Test
+    public void itShouldReturnTwoPairs() {
```

Comment:
It would be great to add negative tests as well for all your test classes. That is, for example, when result returned is not true or the value is not the one that matches with rule defined